ldi-api/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/config/LdioInputConfigurator.java

```diff
@@ -0,0 +1,8 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldi.config;
+
+import be.vlaanderen.informatievlaanderen.ldes.ldi.services.ComponentExecutor;
+import be.vlaanderen.informatievlaanderen.ldes.ldi.types.LdiAdapter;
+
+public interface LdioInputConfigurator {
```

Comment:
Do we need an input configurator? Currently the ldesClient does not need to be a bean as it runs with an executor and the httpIn registers its own beans. Maybe a 'void starter' method would be more accurate and natural to understand when developping a new input component?